apis/falcon/v1alpha1/falconcontainer_types.go

```diff
@@ -1,53 +1,123 @@
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-type FalconContainerInjectorSpec struct {
-	// Define annotations that will be passed down to injector service account. This is useful for passing along AWS IAM Role or GCP Workload Identity.
-	SAAnnotations map[string]string `json:"sa_annotations,omitempty"`
-}
 
 // FalconContainerSpec defines the desired state of FalconContainer
+// +k8s:openapi-gen=true
 type FalconContainerSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// FalconAPI configures connection from your local Falcon operator to CrowdStrike Falcon platform.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Platform API Configuration",order=1
 	FalconAPI FalconAPI `json:"falcon_api"`
+
 	// Registry configures container image registry to which the Falcon Container image will be pushed
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Image Registry Configuration",order=2
 	Registry RegistrySpec `json:"registry,omitempty"`
 
-	// InstallerArgs are passed directly down to the Falcon Container Installer. Users are advised to consult Falcon Container documentation to learn about available command line arguments at https://falcon.crowdstrike.com/documentation/146/falcon-container-sensor-for-linux
-	InstallerArgs []string `json:"installer_args,omitempty"`
 	// Injector represents additional configuration for Falcon Container Injector
-	Injector *FalconContainerInjectorSpec `json:"injector,omitempty"`
+	// +kubebuilder:default:={imagePullPolicy:Always}
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Injector Configuration",order=3
+	Injector FalconContainerInjectorSpec `json:"injector,omitempty"`
 
-	// Falcon Container Version. The latest version will be selected when version specifier is missing.
+	// Falcon Container Version Pinning. If not set to false, once a sensor version is set, it is used until manually adjusted.
+	// +kubebuilder:default:=true
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Falcon Container Image Version Pinning",order=4
+	VersionPinning bool `json:"versionPinning,omitempty"`
```

Comment:
What's the point of this when the version field provides this?

Reply:
Discussed with @redhatrises -- updated attribute name to VersionLocking.  This allows for automatic updates to the sensor version being used, if the version isn't set or doesn't identify a single unique image (e.g., 6.46, rather than 6.46.0-2330).